### DISCUSSION TITLE
model_devi problem

### DISCUSSION THREAD

**Discussion by poyi81** [deepmodeling/dpgen#1357] [Q&A]
Dear everyne,
Here are the errors I encountered. The procedure stop in the model_devi step. 
> Traceback (most recent call last):
  File "/dicos_ui_home/litimmy/.local/bin/dpgen", line 8, in <module>
    sys.exit(main())
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpgen/main.py", line 233, in main
    args.func(args)
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpgen/generator/run.py", line 5109, in gen_run
    run_iter(args.PARAM, args.MACHINE)
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpgen/generator/run.py", line 4451, in run_iter
    run_model_devi(ii, jdata, mdata)
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpgen/generator/run.py", line 1962, in run_model_devi
    run_md_model_devi(iter_index, jdata, mdata)
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpgen/generator/run.py", line 1955, in run_md_model_devi
    submission.run_submission()
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpdispatcher/submission.py", line 257, in run_submission
    self.handle_unexpected_submission_state()
  File "/dicos_ui_home/litimmy/.local/lib/python3.9/site-packages/dpdispatcher/submission.py", line 355, in handle_unexpected_submission_state
    raise RuntimeError(
RuntimeError: Meet errors will handle unexpected submission state.
Debug information: remote_root==/ceph/sharedfs/user/litimmy/graduate/run/work/6aff9ea7ea29c67d9d4d620ae77767b75d0a25ee.
Debug information: submission_hash==6aff9ea7ea29c67d9d4d620ae77767b75d0a25ee.
Please check the dirs and scripts in remote_root. The job information mentioned above may help.

And here is the task000.000* in the remote_root. 
<img width="1422" alt="截圖 2023-10-13 晚上8 20 45" src="https://github.com/deepmodeling/dpgen/assets/143478717/45bdd97e-ea40-4538-9ce6-76eb8d4b8783">
In some of the task000.000* directories the lammps error is "core dumped".
<img width="1440" alt="截圖 2023-10-13 晚上8 29 58" src="https://github.com/deepmodeling/dpgen/assets/143478717/c823d171-bdd9-47a4-a9ed-99563baa64a4">
In some of the task000.000*, it doesn't even start running. Just conf.lmp and input.lammps in there.
In some task000.000*, it just stopped without any errors. For example task.000.000002.
<img width="1433" alt="截圖 2023-10-13 晚上8 25 55" 
src="https://github.com/deepmodeling/dpgen/assets/143478717/c9cd689e-3fe2-4c76-a4a3-adfb1eb214da">
<img width="1326" alt="截圖 2023-10-13 晚上8 54 08" src="https://github.com/deepmodeling/dpgen/assets/143478717/4e12f228-c116-4767-af9c-5445c88cc901">
Here is what is inside the task.000.000002. Out of curiosity, I take the input.lammps, conf.lmp and the graph.00*.pb file to my local machine to run normal lammps.
Here is the input.lammps from task.000.000002.
<img width="934" alt="截圖 2023-10-13 晚上8 38 25" src="https://github.com/deepmodeling/dpgen/assets/143478717/96d2b8cc-267c-4857-87e6-981bd070052b">
Then this error showed up.
<img width="467" alt="截圖 2023-10-13 晚上8 40 15" src="https://github.com/deepmodeling/dpgen/assets/143478717/0bab770c-9b5f-4122-a91c-c6d3ce56ff70">
But If I remove the "if "${restart} > 0" then "read_restart dpgen.restart.*" else ", "if "${restart} == 0" then " and "traj/*.". The lammps can run without error.
<img width="751" alt="截圖 2023-10-13 晚上8 49 20" src="https://github.com/deepmodeling/dpgen/assets/143478717/4652b628-04e9-4acd-87bd-1aa836ba07e6">
<img width="1206" alt="截圖 2023-10-13 晚上8 49 43" src="https://github.com/deepmodeling/dpgen/assets/143478717/7c5b30e4-d276-4619-8dd3-06d68603555d">

Could anyone tell me how to deal with this issue? Is my initial coordinates wrong or any weird thing happened in my training data?
Thanks.

Here is the para.json I use. If you need any related file. Please leave a comment. 
<img width="1104" alt="截圖 2023-10-13 晚上9 25 14" src="https://github.com/deepmodeling/dpgen/assets/143478717/5f11d87a-719d-4750-a621-a8f3efd1db76">
<img width="1014" alt="截圖 2023-10-13 晚上9 25 41" src="https://github.com/deepmodeling/dpgen/assets/143478717/e0064eda-ecae-432a-acdd-306cc9192742">
Thank you so much.

**Comment by njzjz** [comment]
See https://github.com/tensorflow/tensorflow/issues/41532
